### PR TITLE
 i add new file can u checking

### PR DESCRIPTION
**Pull request by asadhameedafridi**

### CODE REVIEW

File: Person.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import java.io.Serializable;
+public class Person implements Serializable{
```
Comment (asadhameed) [inline Person.java:2]: why it serializable ? and where is another class

File: Person.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+	private String fName;
+	private String lName;
+
+	public Person(){
+	fName="";
+	lName="";
+	}
+	
+	public Person(String f, String l){
```
Comment (asadhameed) [inline Person.java:12]: give some name

Comment (asadhameedafridi) [reply Person.java:12]: yeah happ

File: Person.java (hunk 1 above)
Comment (asadhameed) [reply Person.java:2]: u not explain

Comment (asadhameedafridi) [reply Person.java:2]: ok now i awork

**Comment by asadhameed** [comment]
please delete this branch

**Pull request closed without merging**